modules/helfi_media_remote_video/helfi_media_remote_video.module

```diff
@@ -130,7 +137,12 @@ function _helfi_media_remote_video_remote_video_validate(array $form, FormStateI
 
   // Convert the video url if needed.
   if ($video_url) {
-    $converted_url = _helfi_media_remote_video_remote_video_url_handler($video_url);
+    try {
+      $converted_url = _helfi_media_remote_video_remote_video_url_handler($video_url);
+    }
+    catch (\Exception $e) {
```

Comment:
I think we should catch `\Drupal\media\OEmbed\ResourceException` here instead

Reply:
Switched to ResourceException.